gnosis/gp-v2-services
# [DB Migration] Fix undesired column names

The recent merge of #906 and #908 introduced undesired column names to the DB while also making a gap in the flyaway DB migration script file naming convention (using V011 instead of V010 - this happened to make room for #875, which will not likely be merged before this).

Game plan for tomorrow is to revert both of the PRs mentioned above, restore staging DB from backup and then merge this as the corrected culmination of both.

Note that - the visable diff with main at this moment just shows the changes after the two branches are merged, but once they are reverted this should appear as all the code required to make the changes introduced in the above to PRs.

## Test Plan

```
- checkout main
- run postgress, apply schema, run orderbook and front end localy,
- place an order

- checkout this branch
- apply schema
- check your existing order and place another order.
```
